.github/workflows/third-party.yaml

```diff
@@ -0,0 +1,114 @@
+# Build this package- no docker container is required
+# this should be broken up more logically
+name: External component
+# concurrency is specified as we can see race conditions with some of the
+# actions that require updates to helm charts and package.json if multiple repos
+# are trying to build at the same time
+concurrency: build_coordinator
+
+# this workflow is expected to the used via 'uses' tag and not run manually
+on:
+  workflow_call:
+    inputs:
+      org:
+        required: true
+        type: string
+        description: "the name of the private org to ensure we don't run on forks"
+      coordinator:
+        required: true
+        type: string
+        description: "the name of the repo that has helper actions in it"
+    secrets:
+      token:
+        required: true
+        description: "org specific token"
+      ci_username:
+        required: true
+        description: "CI username"
+      ci_user_email:
+        required: true
+        description: "CI user email"
+      ci_github_token:
+        required: true
+        description: "CI github token"
+      ci_access_token:
+        required: true
+        description: "CI access token"
+      docker_registry:
```

Comment:
do we have to have these docker ids because of legacy set environment issues? And I totally realise I should have fixed this in the kpack-build one... But I think we should remove them. and we can just pass empty strings in the strep below. Up to you tho

Reply:
ack-- will merge for now, and perhaps we can address some of these flow issues separately. (The answer to your question is probably yes but will need to unpick :)